src/pages/DetailPage.js

```diff
@@ -0,0 +1,84 @@
+import PropTypes from "prop-types";
+import styled from "styled-components/macro";
+import Heading from "../components/Header";
+import Button from "../components/Button";
+import DividingLine from "../components/DividingLine";
+
+DetailPage.propTypes = {
+  concert: PropTypes.object,
+  onNavigate: PropTypes.func,
+  pageName: PropTypes.string,
+};
+
+export default function DetailPage({ concert, onNavigate, pageName }) {
+  return (
+    <div>
+      <Heading pageName={pageName} /> <DividingLine />
+      <Wrapper>
+        <ImageContainer src={concert.image} alt="" />
+        <ArtistContainer>
+          <span>{concert.artist}</span>
+        </ArtistContainer>
+        <DetailsContainer>
+          <Facts>
+            Location:
+            <br /> Termin:
+            <br /> Tickets:
+            <br /> Preis:
+          </Facts>
+          <FactsData>
+            {concert.location}
+            <br /> {concert.date}
+            <br /> www.eventim.de
+            <br /> {concert.price}
+          </FactsData>
```

Comment:
How about a `description list` ? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl

Reply:
Good input! :) I rethink this part and decided to use a table instead. So I get the wanted two columns on my page.